modules/300-prometheus/docs/FAQ_RU.md

```diff
@@ -651,4 +651,46 @@ spec:
   metricsPath: '/metrics'
 ```
 
+## Как опрашивать дополнительные эндпоинты используя Probe?
+
+Добавьте в namespace, в котором находится Probe, лейбл `prometheus.deckhouse.io/probe-watcher-enabled: "true"`.
+
+Example:
+
+```yaml
```

Comment:
Let's change to the single yaml, like in service Monitor example.

Reply:
fixed